.github/workflows/update-load-generator-image.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow uses GHA to build a Java project with Gradle 'load-generator', and build and push multi-arch docker image of this project to the Amazon ECR.
+
+name: Update Load-Generator Image
+
+# Should execute only one workflow run at a time
+concurrency:
+  group: build-and-push-load-generator-image
```

Comment:
Let's add the [cancel in progress](https://github.com/aws-observability/aws-otel-test-framework/blob/aec36308faa0178ee9a65b4bc4fd6bcbd15caea6/.github/workflows/cdk-deploy.yml#L15) field here also.

Reply:
This will ensure that we don't rely on the default behavior.

Reply:
Updated. Done.